spec/models/rent_spec.rb

```diff
@@ -0,0 +1,58 @@
+require 'rails_helper'
+
+describe Rent do
+  describe '#create' do
```

Comment:
we should be using factories for this.
This whole file could be so much simpler with a rent factory. And it should look similar to the book model spec, with the addition of the from < to validation